### PR TITLE
Fixed attribute not found issues for hf_bert_tokenizer

### PR DESCRIPTION
**Pull request by sayanshaw24**
Missing check for existence of attribute before accessing - added.

Validation
- [x] Ran all tests, all 90 passed.

### CODE REVIEW

File: onnxruntime_extensions/pnp/_nlp.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def create_bert_tokenizer(ctx, name, input_names, output_names, container, opera
         # Unfortunately, there's no specific accessor function on
         # transformers.BertTokenizer to query for strip_accents.
         attrs['strip_accents'] = 1 if hf_bert_tokenizer.init_kwargs.get('strip_accents') else 0
```
Comment (wenbingl) [inline onnxruntime_extensions/pnp/_nlp.py:24]: can you add a check for 'strip_accents' as well?

Comment (sayanshaw24) [reply onnxruntime_extensions/pnp/_nlp.py]: done!

File: onnxruntime_extensions/pnp/_nlp.py [hunk 2]
```diff
@@ -21,8 +21,8 @@ def create_bert_tokenizer(ctx, name, input_names, output_names, container, opera
         attrs = dict(vocab_file=vocab)
         # Unfortunately, there's no specific accessor function on
         # transformers.BertTokenizer to query for strip_accents.
-        attrs['strip_accents'] = 1 if hf_bert_tokenizer.init_kwargs.get('strip_accents') else 0
-        attrs['do_lower_case'] = 1 if hf_bert_tokenizer.do_lower_case else 0
+        attrs['strip_accents'] = 1 if hasattr(hf_bert_tokenizer, 'do_lower_case') and hf_bert_tokenizer.init_kwargs.get('strip_accents') else 0
```
Comment (wenbingl) [inline onnxruntime_extensions/pnp/_nlp.py:24]: should it be 'strip_accents'?

Comment (sayanshaw24) [reply onnxruntime_extensions/pnp/_nlp.py]: Yes, sorry, my mistake good catch. Fixed :)

**Review by wenbingl — DISMISSED** [review]

**Review by wenbingl — APPROVED** [review]

**Pull request merged**